lib/import/admin.js

```diff
@@ -142,17 +150,57 @@ function save_features(feature_collection) {
 }
 
 /**
+ * Get hash of admin to population
+ * mutates `feature_collection`.
+ *
+ * @param{string} country_code - Country Code
+ * @param{array} feature_collection - Feature collection
+ * @return{Promise} Fulfilled when csv is read.
+ */
+function world_pop(country_code, feature_collection) {
+  var path = './data/population/';
+  var file = 'worldpop_' + country_code + '.csv';
+  var stream = fs.createReadStream(path + file);
+  var admin_pop = {};
+  return new Promise(function(resolve) {
+    stream.on('error', function(err) {
+      console.log('No population file. Moving along.', err);
+      resolve([{}, feature_collection]);
+    });
+
+    var csvStream = csv()
+    .on('data', function(data) {
+      admin_pop[data[0]] = parseFloat(data[1]);
+    })
+    .on('end', function() {
+      resolve([admin_pop, feature_collection]);
+    });
+    stream.pipe(csvStream);
+  });
+}
+
+/**
  * Import admins from a geojson file and save as Admin documents.
  *
- * @param{string} country_code - 3166-1 alpha-2 country code.
  * @param{string} file - name of geojson file.
  * @param{string} verbose - Option to display debug
  * @return{Promise} Fulfilled when documents are saved.
  */
-function import_admins(country_code, file, verbose) {
+function import_admins(file, verbose) {
+  var country_code = file.split('/')[file.split('/').length - 2];
```

Comment:
Please restore the country_code argument.

 I **don't** want to have the code rely on the structure of the path in any way.

I know it may seem silly since the path is always the same when we run the import, but I strongly prefer having our import tools fully flexible. For example, if we end up using Azure blob stuff, I don't want to be forced to arrange those files in a particular directory structure, just because this script relies on it.